util/container/containers_nodocker.go

```diff
@@ -7,5 +7,5 @@ import "github.com/DataDog/datadog-agent/pkg/util/docker"
 // GetContainers is the unique method that returns all containers on the host (or in the task)
 // and that other agents can consume so that we don't have to convert all containers to the format.
 func GetContainers() ([]*docker.Container, error) {
-	return *docker.Container{}, nil
+	return make([]*docker.Container, 0), nil
```

Comment:
this can just be nil, nil without any allocation

Reply:
@conorbranagan I did it initially with `nil,nil`; however, there's other code that iterates the return (and crashes).  Needs to be empty (so length is zero) I think.

Reply:
Hm you should be able to iterate on a nil map but maybe there's something weird going on. Not a huge deal either way.